dist/profile/templates/confluence/vhost.conf

```diff
@@ -3003,6 +3003,8 @@ RewriteRule "^/display/JENKINS/CSRF\+Protection$" "https://jenkins.io/doc/book/m
 RewriteRule "^/display/JENKINS/FreeBSD$" "https://jenkins.io/doc/book/installing/#freebsd" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/FreeBSD\+4.9$" "https://jenkins.io/doc/book/installing/#freebsd" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/How\+to\+report\+an\+issue$" "https://www.jenkins.io/participate/report-issue/" [NE,NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS//Installing\+Jenkins$" "https://www.jenkins.io/doc/book/installing/#installing-jenkins" [NE,NC,L,QSA,R=301]
```

Comment:
```suggestion
```
I think as @MarkEWaite mentioned on #1486 you may safely get rid of this rule, as the extra slash ('\\') has been placed mistakenly in the wiki

Reply:
It isn't clear to me yet @krharsh17. Should I keeping use one or two slashes?

Reply:
You should use only one slash @getJV

Reply:
Nice! thank you bro. you helped a lot.